libraries/lib-track/Track.h

```diff
@@ -1307,7 +1307,9 @@ struct TrackListEvent
       ADDITION,
 
       //! Posted when a track has been deleted from a tracklist. Also posted when one track replaces another
-      /*! mpTrack points to the first track after the deletion, if there is one. */
+      /*! mpTrack points to the removed track. It is expected, that track is valid during the event.
```

Comment:
This commit is ok except I think this comment should be clearer.  What is “valid”?

say maybe instead, “mpTrack points to the removed track, if some other shared pointer is preventing its deletion”

Reply:
Hmm, I'm trying to say that track pointer is valid during the event in any case